src/index.ts

```diff
@@ -84,10 +85,17 @@ function createExtensions(document, result, requestID, userAgent) {
     ? fetchLoggerRequestDone(requestID, userAgent)
     : {}
 
+  // Should this introspect xapp token/roles for `email_provider` instead?
+  const isEmailProviderRequest = userAgent.includes("Braze")
+  const emailProviderExtensions = isEmailProviderRequest
```

Comment:
I wonder if here and elsewhere we might want to keep the naming generic, this seems like it could be useful to extend beyond emailProviders in the future e.g. isRateLimitedProviderRequest, rateLimitedProviderExtensions, etc

Reply:
So the GraphQL way wouldn't particularly be to do anything special for a 429 from a backend - just the normal `errors` handling and returning a 200. Although for convenience one _could_ augment `extensions` as needed.

Really - GraphQL clients should be able (hopefully) to introspect the proper response (between `data` and `errors`) and do what they need. It's b/c Braze isn't really a client in our control that we even have to do something custom for it.

Which is to say - as we don't typically support non-Artsy clients hitting MP as part of platform operations - we probably _shouldn't_ offer non-GraphQL-esque patterns unless absolutely necessary.